apiserver/common/action.go

```diff
@@ -103,6 +103,29 @@ func BeginActions(args params.Entities, actionFn func(string) (state.Action, err
 	return results
 }
 
+// ResetActions calls reset on every action passed in through args.
+// It's a helper function currently used by the uniter
+// It needs an actionFn that can fetch an action from state using it's id, that's usually created by AuthAndActionFromTagFn
+func ResetActions(args params.Entities, actionFn func(string) (state.Action, error)) params.ErrorResults {
```

Comment:
Should we call this "RequeueActions"?
It perhaps conveys more meaning as "Reset" is maybe too generic?